assets/js/customizer-preview.js

```diff
@@ -0,0 +1,49 @@
+( function() {
+
+	var backgroundColors = { // TODO: Define in PHP then add as vars to the script.
+		content: {
+			setting: 'background_color',
+			elements: {
+				text: 'body',
+				accent: 'a, .wp-block-button.is-style-outline, .has-drop-cap:not(:focus):first-letter, a.previous-post, a.next-post'
+			}
+		}
+	};
```

Comment:
This should be moved to a `wp_localize_script()` call, both for this file and for the file responsible for the controls. We can then add a filter so users can add their own.